version.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash 
+
```

Comment:
Always use `set -e` in bash/shell scripts to catch possible script errors and stop the execution

Reply:
I do not think it is useful since it is only a single maven command "formatted" to print only the zal version. Anyway, I will change it.